package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/verrazzano/golang:v1.20.12 
 RUN yum update -y \
```

Comment:
You need to add the build of backup-restore-operator to this file.  It was built in the dapper file, but it used golang 1.20.10. 

You also need to change the COPY statements to copy the build results from the ghcr.io/verrazzano/golang:v1.20.12 image.

You want to do something similar to what is happening in this file: https://github.com/verrazzano/rancher/blob/oracle/release/2.7.8/package/Dockerfile#L4

Reply:
Thank you for letting me know. I have made the changes accordingly. Build is passing now.